nullaway/src/test/java/com/uber/nullaway/NullAwayVarargsTests.java

```diff
@@ -46,13 +46,31 @@ public void testNullableVarargs() {
             "public class Utilities {",
             " public static String takesNullableVarargs(Object o, @Nullable Object... others) {",
             "  String s = o.toString() + \" \";",
-            "  // BUG: Diagnostic contains: enhanced-for expression others is @Nullable",
+            "  // BUG: Diagnostic contains: enhanced-for expression others is @Nullable", // Shouldn't be an error!
             "  for (Object other : others) {",
             "    s += (other == null) ? \"(null) \" : other.toString() + \" \";",
             "  }",
+            "  // BUG: Diagnostic contains: enhanced-for expression others is @Nullable", // Shouldn't be an error!
+            "  for (Object other : others) {",
+            "    s += other.toString();", // SHOULD be an error!
+            "  }",
             "  return s;",
             " }",
             "}")
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "public class Test {",
+            "  public void testNonNullVarargs(Object o1, Object o2, Object o3, @Nullable Object o4) {",
+            "    Utilities.takesNullableVarargs(o1, o2, o3, o4);",
+            "    Utilities.takesNullableVarargs(o1);", // Empty var args passed
+            "    Utilities.takesNullableVarargs(o1, o4);",
+            "    Utilities.takesNullableVarargs(o1, (java.lang.Object) null);",
+            // SHOULD be an error!
+            "    Utilities.takesNullableVarargs(o1, (java.lang.Object[]) null);",
```

Comment:
Turns out there _is_ a way to pass null varargs array from Java 😅

Reply:
Ah, interesting.  Didn't think of this one